lib/go-tc/totest/deliveryservices.go

```diff
@@ -23,19 +23,19 @@ import (
 	"strconv"
 	"testing"
 
+	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/lib/go-util/assert"
-	toclient "github.com/apache/trafficcontrol/traffic_ops/v4-client"
+	toclient "github.com/apache/trafficcontrol/traffic_ops/v5-client"
 )
 
 func CreateTestDeliveryServices(t *testing.T, cl *toclient.Session, td TrafficControl) {
 	for _, ds := range td.DeliveryServices {
-		ds = ds.RemoveLD1AndLD2()
-		if ds.XMLID == nil {
+		if &ds.XMLID == nil {
```

Comment:
This condition should be `ds.XMLID == ""`

Reply:
done